src/restic/snapshot_filter_test.go

```diff
@@ -235,57 +164,58 @@ var expireTests = []restic.ExpirePolicy{
 	{Yearly: 10},
 	{Daily: 7, Weekly: 2, Monthly: 3, Yearly: 10},
 	{Tags: []string{"foo"}},
+	{Tags: []string{"foo", "bar"}},
 }
 
 func TestApplyPolicy(t *testing.T) {
 	for i, p := range expireTests {
-		keep, remove := restic.ApplyPolicy(testExpireSnapshots, p)
+		t.Run(fmt.Sprintf("%d", i), func(t *testing.T) {
```

Comment:
It let me use `Fatalf()` on a per test basis instead of `Errorf()/continue`, but it was not really needed. Reverted.